run_nerf_helpers.py

```diff
@@ -223,7 +221,7 @@ def sample_pdf(bins, weights, N_samples, det=False, pytest=False):
 
     # Invert CDF
     u = u.contiguous()
-    inds = searchsorted(cdf, u, side='right')
+    inds = torch.searchsorted(cdf.detach(), u, right=True)
```

Comment:
Even though method should be ok with tensors with enabled grads, but it seems it is a known bug and one has to detach tensors or set `requires_grad=False` to use this method.
Please see this issue: 
https://github.com/pytorch/pytorch/issues/41389